api/src/main/java/com/debugmafia/clueless/state/Turn.java

```diff
@@ -0,0 +1,6 @@
+package com.debugmafia.clueless.state;
+
+class Turn
```

Comment:
To answer one of the comments made about what parameters should the `Turn` class take - at a bare minimum it _has_  to take a `Player` instance (can't have a turn without a player). In the design document, we also have a `Set` of `BoardLocations` as a parameter...although I can't think at the moment as to why that is.

Reply:
The `Turn` class  will also just be a POJO. No other methods other than getters and setters.